content/en/hosting/kubernetes-vs-docker.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Kubernetes vs Docker
+weight: 3
+description: >
+ Options for installing CHT applications
+---
+There are various options for hosting CHT applications depending on the need and phase of your CHT application. Phases of the CHT application include development, testing, training and deployment. CHT Applications can be deployed on different platforms including: 
+* Local installation on a desktop, laptop or server
+* Self-Hosting/Data center set up
+* Cloud Hosting using a cloud provider (e.g. AWS)
+
+CHT installation can be done using various workflows:
+
+* Docker Compose - This has been available for CHT < 4.x and can been implemented using a docker-compose file. 
+* Kubernetes/k3s installation - CHT installation using k3s k8s distribution is the recommended method for CHT >= 4.x installations.
+
+## Kubernetes/k3s
+
+### K3d CHT Architecture
+
+CHT components have been decomposed into various containers that are orchestrated via docker-compose. However, docker-compose does not provide production grade orchestration. Kubernetes provides various advantages in managing docker containers such as:
+
+* Orchestration of multiple deployments. This ensures that services remain available and any nodes that fail are redeployed and queued.
+* Service discovery: Being able to route public requests to deployments within the cluster 
+* Integration with hypervisors such as VMWare. This is possible due to CRD support in Kubernetes that allows 3rd parties to create integration with k8s.
+* Helm support and integration that makes it easy to easily deploy applications on to the cluster.
+
+The main deployments (components) of a CHT deployment include CHT Sentinel, CHT API, CHT HAProxy Health-check, CHT HAProxy, Upgrade Service and CHT CouchDB Instances.
+
+These components are deployed as k8s pods as:
+
+* A single pod for each couchDB instance.
+* A CHT api pod.
+* A CHT HAProxy Healthcheck pod.
+* A CHT HAProxy pod.
+* Upgrade service pod.
+* CHT-Sentinel pod.
+
```

Comment:
## Docker Compose

The Docker Compose based CHT Core deployment  was Medic's first attempt to make CHT 4.x cloud native.  The Compose files work quite well for application developer setups on laptops and the like (check out the [Docker Helper]({{< relref "hosting/4.x/app-developer#cht-docker-helper-for-4x" >}})!). Additionally, we have existing published guides on how to deploy single and multi-node Compose based solutions.  For small, low use instances, likely the single node Compose deployments will be fine.  We do not recommend setting up a multi-node deployment on Compose with an overlay network.  Please use Kubernetes instead for a more stable and [horizontally scalable]({{< relref "hosting/vertical-vs-horizontal" >}}) solution.

The Compose documentation here is only for reference until Medic is ready to fully deprecate this content.

Like Kubernetes above, Docker Compose deploys the same services but adds an additional 7th to allow for ingress/egress via a reverse proxy:

* A service for each CouchDB instance.
* A CHT API service.
* HAProxy Healthcheck service.
* HAProxy service.
* Upgrade Service service.
* A CHT Sentinel service.
* An nginx service to act as a reverse proxy and terminate TLS connections

Reply:
I see this has been committed - resolving!